frontend/src/views/Support/Unsubscribe.js

```diff
@@ -1,38 +1,125 @@
 import React, {Component} from "react";
 import "../Support/SupportPages.scss";
 import {withRouter} from "react-router-dom";
-import {Container, Row, Button} from "react-bootstrap";
+import {Container, Row, Button, Col} from "react-bootstrap";
 import Navbar from "../../core/components/Navbar";
+import qs from "qs";
+import PocketUserService from "../../core/services/PocketUserService";
+import AppAlert from "../../core/components/AppAlert";
 
 class Unsubscribe extends Component {
-  state = {};
+  constructor(props, context) {
+    super(props, context);
+
+    this.validateSubscribe = this.validateSubscribe.bind(this);
+    this.state = {
+      email: "",
+      alertOverlay: {
+        show: false,
+        variant: "",
+        message: "",
+      },
+    };
+  }
+
+  validateSubscribe() {
+    PocketUserService.subscribeUser(this.state.email).then((result) => {
+      if (result) {
+        this.setState({email: result.data});
+        this.setState({
+          alertOverlay: {
+            show: true,
+            variant: "success",
+            message: "You've been subscribed to our mailing list again.",
+          },
+        });
+      } else {
+        this.setState({
+          alertOverlay: {
+            show: true,
+            variant: "danger", // TODO update this variant to non danger
```

Comment:
fix this TODO or delete if done.

Reply:
I deleted it and committed the code since this wasn't needed anymore.